### PR TITLE
AC Automated Test - LRAC-11443 | master

### PR DESCRIPTION
**Pull request by manuferreira**
https://issues.liferay.com/browse/LRAC-11443 (automation ticket)
https://issues.liferay.com/browse/LRAC-10629 (bug ticket)
Couldn't run on CI because of the issues we are facing so I decided to open the pull request anyway just in case you guys want me to change something.

### CODE REVIEW

**Comment by manuferreira** [comment]
ci:test:sf

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/AnalyticsCloudConnectionTest.testcase [hunk 1]
```diff
@@ -428,6 +428,56 @@ definition {
 		ACDXPSettings.connectDXPtoAnalyticsCloud();
 	}
 
+	@description = "Bug: LRAC-10629 | Automation ID: LRAC-11443 | Test Summary: Sites and Contacts connection status is not changing when renaming the data source name"
+	@priority = "3"
+	test ConnectionStatusNotChangeWhenRenameDataSourceName {
+		task ("Launch AC then go to Data Sources and open the created data source") {
+			var propertyName = ACDXPSettings.connectDXPtoAnalyticsCloud();
```
Comment (ViniLopes20) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/AnalyticsCloudConnectionTest.testcase:435]: Hey @manuferreira, just a small change here, it is not necessary to put the property name in this variable, the macro returns a global value for every test that uses the macro. So to access the property name you can use `${assignedPropertyName}`

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/AnalyticsCloudConnectionTest.testcase [hunk 2]
```diff
@@ -428,6 +428,56 @@ definition {
 		ACDXPSettings.connectDXPtoAnalyticsCloud();
 	}
 
+	@description = "Bug: LRAC-10629 | Automation ID: LRAC-11443 | Test Summary: Sites and Contacts connection status is not changing when renaming the data source name"
+	@priority = "3"
+	test ConnectionStatusNotChangeWhenRenameDataSourceName {
```
Comment (ViniLopes20) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/AnalyticsCloudConnectionTest.testcase:433]: Hey @manuferreira, I think you forgot to do these steps in the test:

1. Disconnected the data source;
2. Observe the Current Status is DISCONNECTED;
3. Rename the Data Source name;
4. Observe the Current Status is DISCONNECTED;
5. Refresh/reload the page;
6. Observe the Current Status is DISCONNECTED;

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/AnalyticsCloudConnectionTest.testcase (hunk 1 above)
Comment (ViniLopes20) [reply portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/AnalyticsCloudConnectionTest.testcase:435]: Yes, I like it. Updates on test and test map. Thank you, Manu

**Comment by manuferreira** [comment]
ci:test:sf

**Review by ViniLopes20 — APPROVED** [review]
LTGM, thanks Manu 🎉

**Review by calvinctvo — APPROVED** [review]
Hey @manuferreira ! The test LGTM! Thank you

**Review by joshitagaki — APPROVED** [review]
Nice job! LGTM!

**Review by christianstokes — APPROVED** [review]
Looks really good, thanks @manuferreira 

And good review comments @ViniLopes20 👏👍

**Comment by ViniLopes20** [comment]
Hi team, Thanks for reviewing!

I'm forwarding it!

**Comment by ViniLopes20** [comment]
ci:forward

**Pull request closed without merging**